include/zenoh-pico/transport/unicast/lease.h

```diff
@@ -0,0 +1,25 @@
+//
+// Copyright (c) 2022 ZettaScale Technology
+//
+// This program and the accompanying materials are made available under the
+// terms of the Eclipse Public License 2.0 which is available at
+// http://www.eclipse.org/legal/epl-2.0, or the Apache License, Version 2.0
+// which is available at https://www.apache.org/licenses/LICENSE-2.0.
+//
+// SPDX-License-Identifier: EPL-2.0 OR Apache-2.0
+//
+// Contributors:
+//   ZettaScale Zenoh Team, <zenoh@zettascale.tech>
+//
+
+#ifndef ZENOH_PICO_UNICAST_LEASE_H
+#define ZENOH_PICO_UNICAST_LEASE_H
+
+#include "zenoh-pico/transport/transport.h"
+
+int8_t _zp_unicast_send_keep_alive(_z_transport_unicast_t *ztu);
+int _zp_unicast_start_lease_task(_z_transport_t *zt, _z_task_attr_t *attr, _z_task_t *task);
+int _zp_unicast_stop_lease_task(_z_transport_t *zt);
```

Comment:
As convention, return `int8_t` instead.
This goes back to a previous discussion on having a list of return codes.